tests/unit/literaturesuggest/test_literaturesuggest_tasks.py

```diff
@@ -94,6 +94,53 @@ def test_new_ticket_context():
     assert expected == result
 
 
+def test_new_ticket_context_handles_unicode():
+    data = {
+        'titles': [
+            {
+                u'title': 'Actual title contained: é',
```

Comment:
Took me a while to find where the data is (_didn't know we could see these_) -  changing that now.